Add `sqltools.disableChordKeybindings` setting to disable Ctrl/Cmd+E chords (#845)

gjsjohnmurray opened the pull request:
This fixes #845. Previously the extension added a set of chord keybindings beginning with` Ctrl/Cmd+E`. As a consequence users suddenly lost the ability to invoke VS Code's `Go to File...` command from the keyboard using its standard keybinding (Ctrl/Cmd+E).

This change adds a new setting `sqltools.disableChordKeybindings` and defaults it to `false`.

Also add the chord `Ctrl/Cmd+E Ctrl/Cmd+E` to invoke the masked command as long as focus is not on selected text in an editor.

While adding the new setting I also polished descriptions for other settings.

----

Thank you for your contribution! 
Before submitting this PR, please make sure:

- [x] Your code builds clean without any errors or warnings
- [x] You have made the needed changes to the docs
- [x] You have written a description of what is the purpose of this pull request above

gjsjohnmurray committed:
Add `sqltools.enableOldKeybindings` setting to disable Ctrl/Cmd+E chord keybindings (#845)

gjsjohnmurray committed:
Remove unused import

mtxr commented:
@gjsjohnmurray the project is quite old and we don't have much reports about this, instead of making this a breaking change, why not make this an opt in setting to disable the chord instead?

Either way, I believe key bindings are easy to change. I started the project for VSCode because Sublime users were migrating (like I was) so I wanted to use my project from there (https://packagecontrol.io/packages/SQLTools) here, for the same reason MS team created a package to make it easier for sublime users to onboard on vscode (https://marketplace.visualstudio.com/items?itemName=ms-vscode.sublime-keybindings), so I have no idea if some users came here and still uses the key bindings I use, but a breaking change is always something that will affect daily work.

As you know, on my daily basis right now i'm not using sqltools often anymore, so would be a big deal to me, but it's still just one report, I believe we should open this for discussion with more developers, just don't know where

gjsjohnmurray committed:
Invert meaning of new keybinding setting
Also rename it `sqltools.disableChordKeybindings`  and have it false by default.

gjsjohnmurray commented:
@mtxr coincidentally I had just reached the same conclusion about not making this a breaking change. I have pushed a commit which inverts the logic of the new (renamed) setting.
